src/frame/auction.js

```diff
@@ -44,7 +44,10 @@ export default async function runAdAuction (conf, debug) {
 	}
 
 	debug && echo.log(echo.asProcess('getting all scores, filtering and sorting'));
-	const [ winner ] = await getScores(filteredBids, conf, debug);
+	const winners = await getScores(filteredBids, conf, debug);
+	const [ winner ] = winners
+		.filter(({ score }) => score > 0)
+		.sort((a, b) => (a.score > b.score) ? 1 : -1);
```

Comment:
A possible future optimization: is filter + sort more expensive than sorting and just checking the highest value for >0?